components/ItemCard/ItemCard.js

```diff
@@ -0,0 +1,88 @@
+import {
+  StyledListItem,
+  StyledArticle,
+  StyledDiv,
+  StyledH2,
```

Comment:
If you style components that are specific to this page, you should try to be more descriptive.
Example: instead of "StyledH2", use "StyledProductTitle" or something like that.
However, if you find this styling should be universal to _all_ your h2's, the styling shouldn't take place in your component

Reply:
I had to rename the components. It's not semantic enough.
Will modify it accordingly.
But all styling are temporary.